data/shared/citizen/scripting/lua/utilities.lua

```diff
@@ -0,0 +1,115 @@
+local fmath = math
+local math = {}
+
+setmetatable(math, {__index = fmath})
+
+-- Beginning of the Switch and case statements
+
+_G.switch = function(pParameter, pCases)
+  if (not pParameter) then return false, "pParameter: Cannot be 'null'" end
+
+  local case = pCases[pParameter]
+  if case then return case() end
+    
+  local define = pCases['default']
+  return define and define() or nil
+end
+
+-- Ending of the Switch and case statements
+
+
+
+-- Beginning of the table-based utilities
+table.print = function(pTable, toPush, noTab)
+	if (not noTab) then print("Table:") end
+  
+	local tabsFound = toPush or 1
+	local tabsCache = "\t";
+  
+	if(tabsFound > 1) then
+		for i = 1, tabsFound do
+			tabsCache = tabsCache .. "\t";
+		end
+	end
+  
+	if(type(pTable) == "table") then
+		for k,v in pairs(pTable) do
+			if(type(v) == "table") then
+				print(tabsCache .. k .. ":");
+				table.print(v, tabsFound + 1, true);
+			else
+				print(tabsCache .. k .. " = "..tostring(v));
+			end
+		end
+	else
+		print(tabsCache .. tostring(pTable))
+	end
+end
+
+table.has = function(pTable, pValue)
+  if (pTable == nil) then return end
+  if (pValue == nil) then return pTable[1] end
+
+  for key, value in ipairs(pTable) do
+    if (value ~= pValue) then
```

Comment:
This condition shouldn't exists, return true if `value == pValue` and at the end of the function return false.